output.h

```diff
@@ -19,6 +19,8 @@ struct cg_output {
 	struct wl_listener destroy;
 	struct wl_listener damage_frame;
 	struct wl_listener damage_destroy;
+
+	struct wl_list link;
```

Comment:
Please add ` // server::outputs` here to indicate which list this is part of.

Reply:
Added `cg_server::outputs` link indication, thanks!